interfaces/cython/cantera/test/test_reactor.py

```diff
@@ -443,7 +443,8 @@ def test_mass_flow_controller(self):
         reservoir = ct.Reservoir(gas2)
 
         mfc = ct.MassFlowController(reservoir, self.r1)
-        mfc.mass_flow_rate = lambda t: 0.1 if 0.2 <= t < 1.2 else 0.0
+        # Triangular pulse with area = 0.1
```

Comment:
While I usually would type it like this also, I believe Cantera uses white spaces left and right of `*`.

Reply:
Fair enough. It doesn't fit on one line anymore, but 🤷 .

Reply:
I think you could get it to fit on a line by reversing the check of the time and returning zero first. Then you could dedent the calculation line

Reply:
Oh, I see now that I'm looking on a computer. Ignore this comment :joy: